yang/ietf-l3vpn-ntw.yang

```diff
@@ -753,38 +753,34 @@ module ietf-l3vpn-ntw {
                   description
                     "An identifier of an active VPN instance profile.";
                 }
+                leaf bearer-reference {
+                    if-feature "vpn-common:bearer-reference";
+                    type string;
+                    description
+                      "This is an internal reference for the service
+                      provider.";
+                  }
                 uses vpn-common:service-status;
                 container connection {
                   description
                     "Encapsulation types.";
-                  leaf encapsulation-type {
-                    type identityref {
-                      base vpn-common:encapsulation-type;
+                  leaf connection-type {
+                    type enumeration {
+                        enum LOCAL {
+                          description
+                            "evpn bgp-instance is operationally up";
+                        }
+                        enum REMOTE {
+                          description
+                            "evpn bgp-instance is operationally down";
+                        }
                     }
                     default "vpn-common:untagged-int";
                     description
                       "Encapsulation type. By default, the encapsulation
                        type is set to 'untagged'.";
                   }
-                  container logical-interface {
-                    description
-                      "Reference of a logical interface
-                       type.";
-                    leaf local-reference {
```

Comment:
We agreed to have this one to address #266

Reply:
reverted 45ca37511f6f7c0b0d19c1f3affe9d742a788e28